instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -517,6 +720,104 @@ def test_wsgi_metrics(self):
                             )
         self.assertTrue(histrogram_data_point_seen and number_data_point_seen)
 
+    # pylint: disable=too-many-locals
+    def test_wsgi_metrics_new_semconv(self):
+        _expected_metric_names = [
+            "http.server.active_requests",
+            "http.server.request.duration",
+        ]
+        _recommended_attrs = {
+            "http.server.active_requests": _server_active_requests_count_attrs_new,
+            "http.server.request.duration": _server_duration_attrs_new,
+        }
+        start = default_timer()
+        for _ in range(3):
+            response = Client().get("/span_name/1234/")
+            self.assertEqual(response.status_code, 200)
+        duration = default_timer() - start
```

Comment:
can we use `duration_s` like we did in other instrumentation (ex., fastapi) ? just to not confuse anyone